tests/flow_test.go

```diff
@@ -2311,7 +2312,7 @@ func testFlowsHashCnx(t *testing.T) *Test {
 			foundAB := false
 			foundBA := false
 			for _, f := range flows {
-				if f.Metric.ABPackets != 1 || f.Metric.BAPackets != 1 {
+				if f.Metric.ABPackets == 0 || f.Metric.BAPackets == 0 {
```

Comment:
we should receive exactly the numbers injected here, so the test should be != 10